pom.xml

```diff
@@ -330,19 +330,19 @@
         </plugin>
         <plugin>
           <artifactId>maven-deploy-plugin</artifactId>
-          <version>2.8.2</version>
+          <version>3.0.0-M1</version>
```

Comment:
https://issues.apache.org/jira/browse/MDEPLOY-244 as @jtnord commented in https://github.com/jenkinsci/pom/pull/37